exampleClientSmithy/smithy/example.smithy

```diff
@@ -0,0 +1,18 @@
+$version: "2.0"
```

Comment:
Can't we make these smithy files local to the same client module as the one that uses the generated code?

Reply:
done